### DISCUSSION TITLE
Show/Hide parts in Paraview

### DISCUSSION THREAD

**Discussion by Serince** [OpenRadioss/OpenRadioss#477] [Q&A] [answered]
The result files are converted very nicely as described. Is it possible to show/hide parts in paraview after converting the result files to vtk and csv?

**Comment by drozp94** [comment, marked as answer]
Hello Serince,
I've observed that vtk file contains PART ID field, so you can use Threshhold filter from Paraview to extract the desired part by it's ID!

Good luck!
Dima

**Reply by nawijn** [reply]
Although you are technically correct, in my opinion it is not enough to easily get the representation that you want. The reason is that a part typically also includes elements representing boundary conditions. For example when selecting a single PART_ID for the deep drawing example, you still get the following:

![image](https://user-images.githubusercontent.com/467031/192808888-695f8d80-d712-4482-90d7-a5d371ce4282.png)

Therefore it would be really nice that during export the physical parts of the model are separated from the `supporting elements`.

Makes sense?

Marco

**Reply by drozp94** [reply]
Hello Marco,

sorry for late response,
I would start to say that you are right about "nice that during export the physical parts of the model are separated from the supporting elements". 

I've did this workaround in another discussion topic. Please check this link: https://github.com/orgs/OpenRadioss/discussions/484#discussion-4401420

![image](https://user-images.githubusercontent.com/113352419/192950785-26f5bd90-4f1e-4260-89de-98f3420c3dd4.png)

Thank you!
Dima
